PROVISIONING-METAL.md

```diff
@@ -34,11 +34,12 @@ sha512sum -c <<<"a3c58bc73999264f6f28f3ed9bfcb325a5be943a782852c7d53e803881968e0
 
 Next, set your desired version and variant, and use `tuftool` to download the image:
 To install `tuftool` you'll need to install Rust (via [rustup](https://rustup.rs/) or the official site), and then you can run `cargo install tuftool`.
+The VERSION corresponds to the [Bottlerocket version](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.16.1), the latest release is almost always what you want.
```

Comment:
Rather than linking to v1.16.1, I think it might be better to just go the [releases page](https://github.com/bottlerocket-os/bottlerocket/releases) and let them find it. Your comment about latest release kind of points to that anyway.

Reply:
Oh, good catch, I tried to copy the `latest` url which ended up redirecting me to 1.16.1 and that is the url I copied. `https://github.com/bottlerocket-os/bottlerocket/releases/latest` is what I typed to get there :) I'll update to that which should land the user on the latest release.